CMakeLists.txt

```diff
@@ -240,7 +240,7 @@ macro(xeus_r_create_target target_name linkage output_name)
         set(XEUS_R_XEUS_TARGET xeus-static)
     endif ()
 
-    target_link_libraries(${target_name} PUBLIC ${XEUS_R_XEUS_TARGET} PRIVATE ${R_LIBRARY_BASE})
+    target_link_libraries(${target_name} PUBLIC ${XEUS_R_XEUS_TARGET} PRIVATE ${R_LIBRARY_BASE} ${R_LIBRARY_BLAS} ${R_LIBRARY_LAPACK})
```

Comment:
Could you try with blas only and no lapack?

Reply:
Was on my radar. Next commit.